### PR TITLE
Build: Login

### LINKED ISSUE #33: Log In
resim

Component with functionality

### PR DESCRIPTION
**Pull request by udenizdemirbilek**
Draft login page for review

Signs in test users with email&password authentication, redux is set up and working

To do:
Will add google and facebook login buttons,
Work on styling

### CODE REVIEW

**Review by halitbatur — COMMENTED** [review]
@udenizdemirbilek so far so good

**Comment by udenizdemirbilek** [comment]
add dispatcher to navbar to change state when user logs out, as well as selector to read state and conditionally render login, signup, logout and user profile buttons 
Please take another look @halitbatur

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-chrome",
```
Comment (halitbatur) [inline .vscode/launch.json]: why is this file changed ?

File: package.json [hunk 2]
```diff
@@ -3,18 +3,21 @@
...
     "dependencies": {
+        "@reduxjs/toolkit": "^1.6.1",
         "@testing-library/jest-dom": "^4.2.4",
         "@testing-library/react": "^12.0.0",
         "@testing-library/user-event": "^7.1.2",
         "bootstrap": "4.6.0",
         "bootstrap-icons": "^1.5.0",
-        "firebase": "^8.9.1",
+        "firebase": "8.9.1",
+        "firebaseui": "^5.0.0",
```
Comment (halitbatur) [inline package.json]: I don't think we need this package

File: src/components/Navbar/NavbarForLargeScreen.jsx [hunk 3]
```diff
@@ -1,9 +1,32 @@
...
+import { useSelector, useDispatch } from "react-redux";
+import firebase from "firebase/app";
+import "firebase/auth";
+import { largeScreenNavbarData } from "../../data/Navbar/largeScreenNavbarData.js";
 
 const NavbarForLargeScreen = () => {
+  const { userId, userName } = useSelector((state) => state.user);
+  const displayProfile = userId === "" ? "none" : "inline-block";
+  const displayLogin = userId === "" ? "inline-block" : "none";
+  const dispatch = useDispatch();
```
Comment (halitbatur) [inline src/components/Navbar/NavbarForLargeScreen.jsx:13]: new line after this

File: src/containers/Login/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+          <Form.Group controlId="formBasicEmail">
+            <Form.Label>Email address</Form.Label>
+            <Form.Control
+              type="email"
+              placeholder="Enter email"
+              onChange={loginForm}
+              value={login.email}
+            />
+          </Form.Group>
+
```
Comment (halitbatur) [inline src/containers/Login/Login.jsx]: extra new line

**Review by halitbatur — CHANGES_REQUESTED** [review]

**Comment by halitbatur** [comment]
@udenizdemirbilek good work Deniz! I have written some changes, and u will need to update the branch and resolve conflicts

File: .vscode/launch.json (hunk 1 above)
Comment (udenizdemirbilek) [reply .vscode/launch.json]: I think this file was missing and I tried to run vscode debugger a few days ago, and it created this file, I can either delete it or add it to gitignore

File: package.json (hunk 2 above)
Comment (udenizdemirbilek) [reply package.json]: firebaseui removed!

**Review by ceyhungulbas — APPROVED** [review]
LGTM! 🎉

**Review by halitbatur — APPROVED** [review]
